next.config.mjs

```
@@ -1,10 +1,19 @@
+import withBundleAnalyzerFn from '@next/bundle-analyzer';
 import { withSentryConfig } from '@sentry/nextjs';
 import path from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)